src/helpers.ts

```diff
@@ -91,3 +92,35 @@ export const checkPackageUpdates = async (
     // Cannot verify version, might be network error etc. We don't bother showing anything in that case
   }
 }
+
+export const convertStepToRoute = (step: Step): Route | null => {
```

Comment:
This will never return `null`.

Reply:
I'm trying to put a validation check on `estimate.toAmountUSD` and `estimate.fromAmountUSD` which is optional but required in Route.
So, if this field is `undefined` then throw an error.

and since this function is expecting something to be returned, I've returned null in case of error.

Hence, the 2 step validation.

Would it be recommended that if these two values are undefined, we take it as 0?

Reply:
> and since this function is expecting something to be returned, I've returned null in case of error.

@Abhikumar98 It doesn't work like that 🙂  Please check the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw) 🙂 

> The throw statement throws a user-defined exception. Execution of the current function will stop (the statements after throw won't be executed), and control will be passed to the first [catch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) block in the call stack. If no catch block exists among caller functions, the program will terminate.

Reply:
Yes, this worked now.
surprisingly it was showing an error earlier about expecting a value to be returned.

Thanks @chybisov